ci-operator/step-registry/ipi/conf/aws/usage-info/ipi-conf-aws-usage-info-ref.yaml

```diff
@@ -0,0 +1,19 @@
+ref:
+  as: ipi-conf-aws-usage-info
+  from_image:
+    namespace: ocp
+    name: "4.16"
+    tag: upi-installer
+  commands: ipi-conf-aws-usage-info-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+  - name: USAGE_CLUSTER_TYPE
+    default: "classic"
+    documentation: |-
+      Indicate this cluster is a classic cluster, or hypershift-mgmt cluster
```

Comment:
should we list all the supported cluster_type here?

Reply:
No, not each type of cluster is supposed to use this step, roas, hypershift-hosted cluster is not supposed to use this step, because they are not using userTags in install-config.yaml.